src/components/tag/filter-tag.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2019
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, property, customElement, LitElement } from 'lit-element';
+import classnames from 'classnames';
+import settings from 'carbon-components/es/globals/js/settings';
+import Close16 from '@carbon/icons/lib/close/16';
+import FocusMixin from '../../globals/mixins/focus';
+import TAG_TYPE from './types';
```

Comment:
Great to have this types re-exported from this file.

Reply:
exported `TAG_TYPE` from `tag.ts` and `filter-tag.ts` and removed some `./tag` imports, let me know if the imports and exports look correct now